lib/mix/tasks/phx.gen.channel.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Mix.Tasks.Phoenix.Gen.Channel do
+  @shortdoc "Generates a Phoenix channel"
+
+  @moduledoc """
+  Generates a Phoenix channel.
+
+      mix phoenix.gen.channel Room
+
+  Accepts the module name for the channel
+
+  The generated model will contain:
+
+    * a channel in web/channels
```

Comment:
it should be `lib/web/channels`. I can fix this (and the mix task name I mentioned above) in a PR later this evening.